nginx/public/node/frontend/public/js/app/search/chant-search/AboutVolpianoView.js

```diff
@@ -0,0 +1,8 @@
+import _ from "underscore";
```

Comment:
What is going on here?

Reply:
Nope....a wayward and unedited copy-paste situation...thanks!